### DISCUSSION TITLE
Better legend for Pie Chart

### DISCUSSION THREAD

**Discussion by mrspartak** [mantinedev/mantine#5763] [Feature requests]
Currently, labels will show value instead of name, which is not helpful at all. It would be cool to have a formatting option + a way to show the percentage rather than the actual value.
![image](https://github.com/mantinedev/mantine/assets/993910/129a59f4-ec12-432e-a1ed-87a6ef7e2cd3)

**Comment by wyatttroutman** [comment]
It looks like the percentage formatting is now an option. There is still missing functionality for the chart legend. The PieChart component does not have the `withLegend` property that the other charts have. There is currently no way shown in the documentation to have a Legend added to your Pie Chart, like in other chart options, without using the tooltip functionality. For dashboards where we want to show simple information at a glance, requiring the tooltip interaction to see the chart legend is not an ideal user experience. 

I looked at the underlying recharts component and it appears to support Legends ([Legend](https://recharts.org/en-US/api/Legend) component), as PieChart is a parent component of their Legend component. 

I would be glad to work on adding support for this, assuming this was just an oversight / pending enhancement and not skipped due to technical issues or complexity.

**Reply by rtivital** [reply]
You are welcome to submit a PR with this feature for PieChart and DonutChart components.
